docs/integration/js-example.md

```diff
@@ -1,9 +1,154 @@
+
+#### On the frontend
+
+```typescript
+import {getSDK} from '@ultimatearcade/client-sdk'
+const uaSDK = getSDK()
+const si = await uaSDK.getSessionInfo()
+// this is the address at which you'll reach your backend server, e.g. `your.host.com:1234`
+const serverAddress = si.server_address
+// this is a secret string that authenticates the user. Send it to your backend so it can use it later
+const = si.player_token
+```
+
+### Ensuring the connected player is valid
+
+When you get a connection (probably via WebSockets) you still need to ensure that a legitimate user has reached your server.
+
+The frontend should therefore send you the `player_token` it got above inside the very first message to your server. You then can verify it and get more player information like this:
+
+```typescript
+const playerInfo = await uaSDK.activatePlayer(initialMessage.player_token)
+const playerName = actResp.display_name
+const playerID = actResp.player_id
+```
+
+You MUST use the playerID from this call to identify the player to ensure that the same player can't connect twice. We try to prevent this on our end, too, but there are always some small time windows when this might happen.
+
+You should also not call this method twice. So it's a good idea to store the `player_token` in your player object, and if you get a connection check if it's just a reconnect after a dropped connection by comparing the player tokens.
+
+### Starting the game, locking the pool
+
+:::note
+This is only relevant for pool games. Pool games are games that follow the pattern of
+1. waiting for enough players
+2. starting the game
+3. playing and determining a winner
+4. closing the game
+:::
+
+After all/enough players have joined we need to close the money pool so no new players cannot join in the middle of the running game:
+
+```typescript
+await uaSDK.lockPool()
+```
+
+### Reporting game results
+
+Figuring out who won/lost (in pool or PvP games) or what the score (in leaderboard games) of a player is of course completely game-specific and up to you make sure the mechanics for this are good and fair.
+
+But once you've determined a game result, here's what to do with that information
+
+#### Player lost against an opponent
+
+
+```typescript
+await uaSDK.playerDefeated(loser.player_token, winner.player_token)
+```
+
+#### Player killed themselves
+
+This is only relevant if your game is player competing against each other (like in pvp and pool games) but also has some game level obstacles.
+E.g. in Cyber Worms it's possible to run into a wall and die.
+
+```typescript
+await uaSDK.playerSelfDefeat(loser.player_token)
+```
+
+It depends on the game mode what happens with the player's tokens. In PvP-Mode they lose a part of it as commission and get back the rest. In Pool-Mode they just lose all the money to the pool and the only difference to loosing against a player is in the statistics.
+
+#### Ending the game with a winner
+
+:::note
+This is only relevant for pool games
+:::
```

Comment:
If this is a pool game tutorial, and we've clearly stated that, then we can remove these

Reply:
we'll be pretty hands on with integrations anyway

Reply:
hmm, true, TBH I forgot that we already did basically the same thing in the code-adaptions... I'll think how to unify or better differentiate the two